packages/cli/src/lib/cmds/help.ts

```diff
@@ -34,6 +34,7 @@ const HELP_MESSAGES = new Map<string, string>(
         'updateConfig ........ sets the paths of the jdk or the androidSdk to the given paths',
         'doctor ............. checks that the jdk and the androidSdk are in place and at the' +
             ' correct version',
+        'merge ........ merges your web manifest into twaManifest.json',
```

Comment:
Could you add enough `.` so that the description is aligned with the cases above? (`doctor` is also misaligned)